Makefile

```diff
@@ -54,17 +59,23 @@ usbrelay.c libusbrelay.c: gitversion.h
 
 clean:
 	rm -f usbrelay
-	rm -f libusbrelay.so
+	rm -f libusbrelay.so.$(USBLIBVER)
+	rm -f libusbrelay.so*
 	rm -f gitversion.h
 	$(MAKE) -C usbrelay_py clean
 
 
 install: usbrelay libusbrelay.so
 	install -d $(DESTDIR)$(LIBDIR)
-	install -m 0755 libusbrelay.so $(DESTDIR)$(LIBDIR)
-	install -d $(DESTDIR)$(PREFIX)/bin
-	install -m 0755 usbrelay $(DESTDIR)$(PREFIX)/bin
-
+	install -m 0755 libusbrelay.so.$(USBLIBVER) $(DESTDIR)$(LIBDIR)
+	$(LDCONFIG) -n $(DESTDIR)$(LIBDIR)
+	( cd $(DESTDIR)$(LIBDIR) ;ln -sr libusbrelay.so.$(USBLIBVER) libusbrelay.so )
```

Comment:
I'm not sure this is a good idea. It would be useful if we want to use old `usbrelay` binary with a new `libusbrelay`, but this can cause problems in the future. I'd simply drop this line.